index.css

```diff
@@ -1,13 +1,129 @@
-.hero {
-  background-image: url("./assets/01_Main_Hero_Mobile.jpg");
-  background-size: cover;
-  background-position: center 33%;
-  height: 100vh;
+
+
+/*////////////////// NAV-BAR*/
+.nav-bar{
+  padding: 0 10px ;
   width: 100%;
+  height: 100px;
+  position: fixed;
+  top: 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  }
+  li,h2{
+    color: blanchedalmond;
+  }
+  .nav-container{
+    flex-basis: 50%;
+    display: none;
+    justify-content: space-between;
+
+  }
+  .nav-menu{
+    flex-basis: 40%;
+    display: flex;
+    justify-content: space-around;
+    padding-right: 10px;
+  }
+  .logo{
+    flex-basis: 10%;
+  }
+  
+
+  .logo, li{
+    font-size: 20px;
+    font-weight: bold;
+    list-style: none;
+  }
+
+/*//////////////////// main sections*/
+  .hero {
+    background-image: url("./assets/01_Main_Hero_Mobile.jpg");
+    background-size: cover;
+    background-position: center 33%;
+    height: 100vh;
+    width: 100%;
+    display: grid;
+    
+    grid-template-columns:1fr 1fr 1fr 1fr 1fr;
+    grid-template-rows: 100px 3fr 1fr 1fr;
+    grid-template-areas: 
+    "b b b b b"
+    ". c c c ."
+    ". d d d ."
+    ". . e . .";
```

Comment:
in a real world, the grid-template-areas should have more valuable names (`navbar` instead of `b`) ;)